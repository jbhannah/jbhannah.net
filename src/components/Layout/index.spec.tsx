import { render, screen } from "@testing-library/react"
import * as React from "react"
import { Layout } from "."

describe("Layout", () => {
  beforeEach(() => render(<Layout data-testid="layout" />))

  it("renders correctly", () => {
    expect(screen.getByTestId("layout")).toMatchSnapshot()
  })
})
