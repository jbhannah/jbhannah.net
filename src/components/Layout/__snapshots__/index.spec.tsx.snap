// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
<div
  class="css-1us33nt-Layout"
>
  <header
    class="css-dm2l2k-Header"
  >
    <h1
      class="css-rrosgf-Header"
      css="[object Object]"
    >
      <a
        aria-current="page"
        class="css-1qnptjq-Header"
        css="[object Object]"
        href="/"
      >
        <picture
          class="css-o08s1i-Header"
        >
          <source
            srcset="avatar-srcSet.webp"
            type="image/webp"
          />
          <img
            alt="Test Site"
            src="avatar.png"
          />
        </picture>
        Test Site
      </a>
    </h1>
    <nav
      class="css-lx2ox9-SocialNav"
    >
      <ul
        class="css-1n7px8z-SocialNav"
      >
        <li
          class="css-h61ul7-SocialLink"
        >
          <a
            class="css-q49akx-SocialLink"
            css="[object Object]"
            href="https://www.example.com"
            rel="noopener"
            target="_blank"
            title="Test Name on Test Service"
          >
            <test-file-stub
              classname="css-78d5kl-SocialLink"
            />
          </a>
        </li>
      </ul>
    </nav>
  </header>
  <main
    class="css-1yh2z2k-Layout"
  />
  <footer
    css="[object Object]"
  >
    <div>
      Copyright Â© 2021 
      <a
        aria-current="page"
        class=""
        css="[object Object]"
        href="/"
      >
        Test Site
      </a>
      . 
      <a
        css="[object Object]"
        href="https://creativecommons.org/licenses/by-sa/4.0/"
        rel="license noopener"
        target="_blank"
      >
        Some Rights Reserved
      </a>
      . 
      <a
        css="[object Object]"
        href="https://github.com/jbhannah/jbhannah.net"
        rel="noopener"
        target="_blank"
      >
        Source on GitHub
      </a>
      .
    </div>
  </footer>
</div>
`;
